codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/parsers/XmlBindingTraitParserGenerator.kt

```diff
@@ -51,9 +51,18 @@ import software.amazon.smithy.rust.codegen.util.outputShape
 import software.amazon.smithy.rust.codegen.util.toPascalCase
 import software.amazon.smithy.rust.codegen.util.toSnakeCase
 
+typealias OperationInnerWriteable = RustWriter.(String) -> Unit
```

Comment:
what is the argument here? looks like the name of the decoder? would be good to document

Reply:
Good point, this is definitely unclear. I will make a follow up PR with a comment and also fixing #417 (since that's driving me bonkers).